package.json

```diff
@@ -45,16 +45,18 @@
     "bunyan-logentries": "^0.1.0",
     "csv": "^1.1.0",
     "debug": "^2.2.0",
+    "hoek": "^5.0.3",
```

Comment:
@Babanila and @LEQADA is it really needed to add `hoek` as a dependency here? As far as I understood @LEQADA's comment we just needed to update `request` and possibly `coveralls` and `sphere-node-sdk`?

Reply:
No, it's not needed. As well as adding `request` package as a dependency. 
And as I previously said it looks redundant to me to create a PR for each 48 vulnerabilities.